bindings/r/DESCRIPTION

```diff
@@ -18,6 +18,7 @@ Suggests:
 Config/testthat/edition: 3
 Config/build/bootstrap: TRUE
 Encoding: UTF-8
+SystemRequirements: C11
```

Comment:
This doesn't do anything. Changing it to `USE_C17` sets `-std=gnu17` which is fine.